GCC 10 incompatibility: radsecproxy.h:265: multiple definition of `pthread_attr'

Trying to build radsecproxy 1.8.1 using gcc-10.0.1 on Fedora Rawhide (for Fedora 32) fails as follows (given it worked with gcc-9.2.1, I assume this is gcc related):

```
Executing(%prep): /bin/sh -e /var/tmp/rpm-tmp.KHxClC
+ umask 022
+ cd /builddir/build/BUILD
+ gpgv2 --keyring /builddir/build/SOURCES/gpgkey-210FA7FB28E45779777BAA1C5963D59C3D68633B.gpg /builddir/build/SOURCES/radsecproxy-1.8.1.tar.gz.asc /builddir/build/SOURCES/radsecproxy-1.8.1.tar.gz
gpgv: Signature made Mon Sep 30 14:33:54 2019 UTC
gpgv:                using RSA key 210FA7FB28E45779777BAA1C5963D59C3D68633B
gpgv: Good signature from "Fabian Mauchle <fabian.mauchle@switch.ch>"
+ cd /builddir/build/BUILD
+ rm -rf radsecproxy-1.8.1
+ /usr/bin/gzip -dc /builddir/build/SOURCES/radsecproxy-1.8.1.tar.gz
+ /usr/bin/tar -xof -
+ STATUS=0
+ '[' 0 -ne 0 ']'
+ cd radsecproxy-1.8.1
+ /usr/bin/chmod -Rf a+rX,u+w,g-w,o-w .
Patch #0 (radsecproxy-1.8.1-paths.patch):
+ echo 'Patch #0 (radsecproxy-1.8.1-paths.patch):'
+ /usr/bin/patch --no-backup-if-mismatch -p1 -b --suffix .paths --fuzz=0
patching file radsecproxy.conf.5.in
+ RPM_EC=0
++ jobs -p
+ exit 0
Executing(%build): /bin/sh -e /var/tmp/rpm-tmp.DkZSFD
+ umask 022
+ cd /builddir/build/BUILD
+ cd radsecproxy-1.8.1
+ CFLAGS='-O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection'
+ export CFLAGS
+ CXXFLAGS='-O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection'
+ export CXXFLAGS
+ FFLAGS='-O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -I/usr/lib64/gfortran/modules'
+ export FFLAGS
+ FCFLAGS='-O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -I/usr/lib64/gfortran/modules'
+ export FCFLAGS
+ LDFLAGS='-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld'
+ export LDFLAGS
+ LT_SYS_LIBRARY_PATH=/usr/lib64:
+ export LT_SYS_LIBRARY_PATH
+ '[' 1 = 1 ']'
+++ dirname ./configure
++ find . -name config.guess -o -name config.sub
+ for i in $(find $(dirname ./configure) -name config.guess -o -name config.sub)
++ basename ./build-aux/config.guess
+ '[' -f /usr/lib/rpm/redhat/config.guess ']'
+ /usr/bin/rm -f ./build-aux/config.guess
++ basename ./build-aux/config.guess
+ /usr/bin/cp -fv /usr/lib/rpm/redhat/config.guess ./build-aux/config.guess
'/usr/lib/rpm/redhat/config.guess' -> './build-aux/config.guess'
+ for i in $(find $(dirname ./configure) -name config.guess -o -name config.sub)
++ basename ./build-aux/config.sub
+ '[' -f /usr/lib/rpm/redhat/config.sub ']'
+ /usr/bin/rm -f ./build-aux/config.sub
++ basename ./build-aux/config.sub
+ /usr/bin/cp -fv /usr/lib/rpm/redhat/config.sub ./build-aux/config.sub
'/usr/lib/rpm/redhat/config.sub' -> './build-aux/config.sub'
+ '[' 1 = 1 ']'
+ '[' x '!=' 'x-Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld' ']'
++ find . -name ltmain.sh
+ ./configure --build=x86_64-redhat-linux-gnu --host=x86_64-redhat-linux-gnu --program-prefix= --disable-dependency-tracking --prefix=/usr --exec-prefix=/usr --bindir=/usr/bin --sbindir=/usr/sbin --sysconfdir=/etc --datadir=/usr/share --includedir=/usr/include --libdir=/usr/lib64 --libexecdir=/usr/libexec --localstatedir=/var --sharedstatedir=/var/lib --mandir=/usr/share/man --infodir=/usr/share/info
checking build system type... x86_64-redhat-linux-gnu
checking host system type... x86_64-redhat-linux-gnu
checking target system type... x86_64-redhat-linux-gnu
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for x86_64-redhat-linux-gnu-gcc... no
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking whether make supports the include directive... yes (GNU style)
checking dependency style of gcc... none
checking for x86_64-redhat-linux-gnu-ranlib... no
checking for ranlib... ranlib
checking for mallopt... yes
checking for nettle_sha256_init in -lnettle... yes
UDP transport enabled
TCP transport enabled
TLS (RadSec) transport enabled
DTLS transport enabled
checking for OpenSSL... yes
OpenSSL found in /usr
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating radsecproxy.1
config.status: creating radsecproxy.conf.5
config.status: creating Makefile
config.status: creating tests/Makefile
config.status: executing depfiles commands
+ /usr/bin/make -O -j6 V=1 VERBOSE=1
Making all in tests
make[1]: Nothing to be done for 'all'.
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o catgconf.o catgconf.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o fticks_hashmac.o fticks_hashmac.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o list.o list.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o debug.o debug.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o hash.o hash.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o main.o main.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o util.o util.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o radsecproxy-hash.o radsecproxy-hash.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o gconfig.o gconfig.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o radmsg.o radmsg.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o hostport.o hostport.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o fticks.o fticks.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o dtls.o dtls.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o tcp.o tcp.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o rewrite.o rewrite.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o tlv11.o tlv11.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o tls.o tls.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection  -Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -o radsecproxy-hash radsecproxy-hash.o fticks_hashmac.o hash.o list.o -lnettle 
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection  -Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -o radsecproxy-conf catgconf.o debug.o gconfig.o util.o  -lnettle 
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o tlscommon.o tlscommon.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o udp.o udp.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -DPACKAGE_NAME=\"radsecproxy\" -DPACKAGE_TARNAME=\"radsecproxy\" -DPACKAGE_VERSION=\"1.8.1\" -DPACKAGE_STRING=\"radsecproxy\ 1.8.1\" -DPACKAGE_BUGREPORT=\"https://radsecproxy.github.io\" -DPACKAGE_URL=\"\" -DPACKAGE=\"radsecproxy\" -DVERSION=\"1.8.1\" -DHAVE_MALLOPT=1 -DHAVE_LIBNETTLE=1 -DUSE_OPENSSL=1 -I.  -DSYSCONFDIR=\"/etc\"  -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -c -o radsecproxy.o radsecproxy.c
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
rm -f librsp.a
ar cru librsp.a debug.o dtls.o fticks.o fticks_hashmac.o gconfig.o hash.o hostport.o list.o radmsg.o radsecproxy.o rewrite.o tcp.o tls.o tlscommon.o tlv11.o udp.o util.o 
ranlib librsp.a
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
make[1]: Entering directory '/builddir/build/BUILD/radsecproxy-1.8.1'
gcc -g -Wall -Werror -fno-strict-aliasing -I/usr/include -I/usr/include/openssl -Wall -pedantic -Wno-long-long -pthread -DRADPROT_UDP -DRADPROT_TCP -DRADPROT_TLS -DRADPROT_DTLS -O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector-strong -grecord-gcc-switches -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -L/usr/lib  -Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld -o radsecproxy main.o librsp.a -lssl -lcrypto -lnettle 
make[1]: Leaving directory '/builddir/build/BUILD/radsecproxy-1.8.1'
/usr/bin/ld: librsp.a(tcp.o):/builddir/build/BUILD/radsecproxy-1.8.1/radsecproxy.h:265: multiple definition of `pthread_attr'; librsp.a(radsecproxy.o):/builddir/build/BUILD/radsecproxy-1.8.1/radsecproxy.h:265: first defined here
/usr/bin/ld: librsp.a(tls.o):/builddir/build/BUILD/radsecproxy-1.8.1/radsecproxy.h:265: multiple definition of `pthread_attr'; librsp.a(radsecproxy.o):/builddir/build/BUILD/radsecproxy-1.8.1/radsecproxy.h:265: first defined here
/usr/bin/ld: librsp.a(tlscommon.o):/builddir/build/BUILD/radsecproxy-1.8.1/radsecproxy.h:265: multiple definition of `pthread_attr'; librsp.a(radsecproxy.o):/builddir/build/BUILD/radsecproxy-1.8.1/radsecproxy.h:265: first defined here
/usr/bin/ld: librsp.a(udp.o):/builddir/build/BUILD/radsecproxy-1.8.1/radsecproxy.h:265: multiple definition of `pthread_attr'; librsp.a(radsecproxy.o):/builddir/build/BUILD/radsecproxy-1.8.1/radsecproxy.h:265: first defined here
/usr/bin/ld: librsp.a(dtls.o):/builddir/build/BUILD/radsecproxy-1.8.1/radsecproxy.h:265: multiple definition of `pthread_attr'; librsp.a(radsecproxy.o):/builddir/build/BUILD/radsecproxy-1.8.1/radsecproxy.h:265: first defined here
/usr/bin/ld: librsp.a(fticks.o):/builddir/build/BUILD/radsecproxy-1.8.1/radsecproxy.h:265: multiple definition of `pthread_attr'; librsp.a(radsecproxy.o):/builddir/build/BUILD/radsecproxy-1.8.1/radsecproxy.h:265: first defined here
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:594: radsecproxy] Error 1
make: *** [Makefile:744: all-recursive] Error 1
error: Bad exit status from /var/tmp/rpm-tmp.DkZSFD (%build)
```

--- a/radsecproxy.c
+++ b/radsecproxy.c
@@ -84,6 +84,7 @@ extern int optind;
 extern char *optarg;
 #endif
 static const struct protodefs *protodefs[RAD_PROTOCOUNT];
+pthread_attr_t pthread_attr;
 
 /* minimum required declarations to avoid reordering code */
 struct realm *adddynamicrealmserver(struct realm *realm, char *id);

--- a/radsecproxy.h
+++ b/radsecproxy.h
@@ -263,7 +263,7 @@ int radsrv(struct request *rq);
 void replyh(struct server *server, unsigned char *buf);
 struct addrinfo *resolve_hostport_addrinfo(uint8_t type, char *hostport);
 uint8_t *radattr2ascii(struct tlv *attr); /* TODO: mv this to radmsg? */
-pthread_attr_t pthread_attr;
+extern pthread_attr_t pthread_attr;
 
 /* Local Variables: */
 /* c-file-style: "stroustrup" */

